uktrade/data-hub-frontend
# Add event name filter to new events collection list

## Description of change
This PR adds an Event Name filter to the new Events Collection page. 
It works by modifying the query that is sent to Activity Stream. 

## Test instructions
- In Django dev API, add the user feature flag user-event-activities to your adviser profile.
- Bring up this branch on your local dev frontend and navigate to http://localhost:3000/events
- You should see an Event name filter on the left hand side. 
- In the filter box type 'project zeus' and press enter. It should add project zeus to the query params in the url and it should only be displaying events called Project Zeus. 
- In the filter box backspace everything and press enter. All the events should reappear. 

## Screenshots
### Before
<img width="1028" alt="image" src="https://user-images.githubusercontent.com/70902973/181228101-834a8377-45e7-4c25-800b-97a4eb5e3baf.png">

### After
<img width="1033" alt="image" src="https://user-images.githubusercontent.com/70902973/181228005-3b43663f-8e49-4def-b89d-fd90d9d6b9f2.png">

## Checklist

[//]: # "When submitting a PR make sure the code review guidelines have been satisfied.
https://github.com/uktrade/data-hub-frontend/blob/master/docs/Code%20review%20guidelines.md"
